### ISSUE TITLE
Upgrade ElasticSearch and Kibiter from 6.1.4 to 6.8.6.

### ISSUE THREAD

**Issue by heming6666** [X-lab2017/grimoirelab-sirmordred#3]
we need to upgrade the version of [kibiter](https://github.com/chaoss/grimoirelab-kibiter) from `6.1.4` to `6.8.6`.

**Comment by WillemJiang** [comment]
I  just did a quick test to upgrade the kibiter from 6.1.4 to 6.8.6.  I got the error that ElasticSearch server need to upgrade to 6.8.x too.
BTW,  we just need to upgrade the docker compose file for it, I don't think it relates to sirmordred.

**Comment by heming6666** [comment]
Yes, we need to run Elasticsearch and Kibiter with matching version numbers. I will upgrade the docker compose file for it and use the secured version.

**Comment by heming6666** [comment]
Here are the docker images for `ElasticSearch 6.8.6` and `Kibiter 6.8.6` from `bitergia`:
- https://hub.docker.com/r/bitergia/elasticsearch
- https://hub.docker.com/r/bitergia/kibiter

The `bitergia/elasticsearch` image is based on `elasticsearch-oss-6.8.6` and `search-guard-6.8.6`. Here's the [Dockerfile](https://github.com/Bitergia/elasticsearch/tree/6.8.6-secured). And the demo users and roles for ES can be found here: https://docs.search-guard.com/6.x-25/demo-users-roles .

**Comment by WillemJiang** [comment]
We need to change the default user and password to secure the ES.
